tests/component_example/valid_component/express_component.yaml

```diff
@@ -0,0 +1,18 @@
+name: Example component
+description: This is an example component
+image: example_component:latest
+
+input_subsets:
+  images:
+    fields:
+      data: binary
```

Comment:
I wrapped this in an object in the manifest, so we can extend it in the future if needed. We might want to do the same for the component spec:

```yaml
input_subsets:
  images:
    fields:
      data:
        type: binary
```

Reply:
good point, i'v added it